poll/translations/en/LC_MESSAGES/text.po

```diff
@@ -0,0 +1,234 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2016-04-07 16:17+0500\n"
+"PO-Revision-Date: 2016-04-07 16:17+0500\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.7\n"
+"X-Poedit-Basepath: ../../..\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Poedit-SearchPath-0: poll.py\n"
+
+#: poll.py
+msgid "Whether or not to display results to the user."
+msgstr ""
+
+#: poll.py
+msgid "The maximum number of times a user may send a submission."
+msgstr ""
+
+#: poll.py
+msgid "Number of times the user has sent a submission."
+msgstr ""
+
+#: poll.py
+msgid "Text to display after the user votes."
+msgstr ""
+
+#: poll.py
+#, python-brace-format
+msgid "'{0}' is not present, or not a JSON array."
```

Comment:
@asadiqbal08 You'll need to regenerate the translation files -- now that you introduced a named placeholder in https://github.com/open-craft/xblock-poll/pull/10/commits/e33ab064fce8ca9cb54fc8ae647ff04b94cabddc, this should be `{field}` instead of `{0}`.

Same comment applies to the corresponding string in the `eo` file.